.github/workflows/integration-test.yml

```diff
@@ -9,7 +9,17 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        ENGINE_VERSION: [5.3.0, 5.6.1, latest]
+        include:
+          - ENGINE_IMAGE: registry.gitlab.com/minetest/minetest/server:5.5.0
+            MTG_VERSION: "5.5.0"
+          - ENGINE_IMAGE: registry.gitlab.com/minetest/minetest/server:5.6.0
+            MTG_VERSION: "5.6.0"
+          - ENGINE_IMAGE: registry.gitlab.com/minetest/minetest/server:5.7.0
+            MTG_VERSION: "5.7.0"
+          - ENGINE_IMAGE: ghcr.io/minetest-hosting/minetest-docker:5.8.0
+            MTG_VERSION: "5.8.0"
+          - ENGINE_IMAGE: ghcr.io/minetest-hosting/minetest-docker:main
+            MTG_VERSION: master
```

Comment:
IMO we only need the `min_minetest_version`, the most recent release, and the latest commit. So 5.6 and 5.7 could be removed to speed up testing.